### DISCUSSION TITLE
Sun and 3D prints

### DISCUSSION THREAD

**Discussion by Landsat-Mike** [STELLA-Landsat/STELLA#33] [Notes from the Field]
If you are using the STELLA out in the sun for a longer period of time the PLA can become soft. Try not to leave it in direct sun for any extended period of time.

**Comment by paulmirel** [comment]
Also, best not to leave it in a hot car. The PLA printing material melts at a pretty low temperature, and thus softens and can change shape at fairly low temperatures too.
